lib/App/Critique/Command/collect.pm

```diff
@@ -42,14 +42,7 @@ sub execute {
 
     my $filter;
     if ( my $f = $opt->filter ) {
-        if ( ref $f eq 'CODE' ) {
-            $filter = $f;
-        }
-        else {
-            $filter = $opt->invert
-                ? sub { $_[0]->stringify !~ /$f/ }
-                : sub { $_[0]->stringify =~ /$f/ };
-        }
+        $filter = file_filter($opt,'file_filter_regex');
     }
```

Comment:
I think this should be renamed to be `file_filter_simple` and I don't think that `file_filter_regex` actually is the default behavior we want (it does more things then the above old version does).